vllm/assets/logs/vllm_tests.yaml

```diff
@@ -0,0 +1,17 @@
+id: "vllm"
+tests:
+ -
+  sample: |-
+    INFO 06-11 21:02:01 cpu_executor.py:72] # CPU blocks: 7281
+  result:
+    custom:
+      level: "INFO"
+      timestamp: 1718132521
+      logger:
+        lineno: 72
+        name: "cpu_executor.py"
+    message: '# CPU blocks: 7281'
+    status: "info"
+    tags:
+     - "source:LOGS_SOURCE"
+    timestamp: 1718132521
```

Comment:
It looks like the timestamp returned from running the tests (`45522121000`) is in the year 3412. I haven't run into this issue before, let me ask my team about it.

Reply:
I am also not sure if we can support testing dates without a year, because whatever timestamp we put here would be wrong next year (when the year implicitly changes, but the log message is the same)

Reply:
This was a problem last time around, I'm honestly a little surprised we're not mocking/seeding the timestamp generation logic.

Reply:
@nhinsch this is the only outstanding issue and I'm afraid I don't have enough context to resolve it. Let me know if I should resend you what Inigo did back in the day as a reference.